app/src/main/java/org/p2p/wallet/newsend/NewSendFragment.kt

```diff
@@ -45,19 +42,18 @@ class NewSendFragment :
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         binding.toolbar.apply {
-            title = recipientUsername ?: recipientAddress
+            title = (recipient as? SearchResult.UsernameFound)?.username
+                ?: recipient.addressState.address.cutMiddle(7)
```

Comment:
maybe extract this logic to presenter and then call setToolbarTitle? 
also this `7` looks nasty, const maybe?